### PR TITLE
fix(endian) Fix filesize handling

### LINKED ISSUE #3: Update lzhuf to a current version with big-endian support
Find the most current version of lzhuf with big-endian support and upgrade to it.

Current version D-Rats is using does not work on big-endian systems.

### PR DESCRIPTION
**Pull request by wb8tyw**
lzhuf.c:
  Filesize must be encoded in little-endian for winlink

### CODE REVIEW

**Comment by wb8tyw** [comment]
This should add the big-endian support.

Next will be to start working on packaging scripts that can be run in GitHub Actions.

**Comment by ka5mye** [comment]
I tested ham-radio-software/lzhuf on my Mac Book Pro.  makefile made executable, it ran and made lzhuf.  I ran John's tests and diffs compared the same.  Works fine on Mac.

**Review by ka5mye — APPROVED** [review]
Compiled and executed on my Mac Book Pro, ran John's tests and diffs showed no differences.

File: lzhuf.c [hunk 1]
```diff
@@ -895,6 +920,22 @@ int Encode (int usock, char *iFile, char *oFile,
    return 0;
 }
 
+static int32 i86ul_to_host(int32 ul)
```
Comment (dslotter) [inline lzhuf.c:923]: I would recommend grouping the two related conversion routines together in the same source file.

**Review by dslotter — COMMENTED** [review]
Other than my one suggestion, looks good.

**Pull request merged**